pkgs/development/python-modules/serverfiles/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "serverfiles";
+  version = "0.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-XhD8MudYeR43NbwIvOLtRwKoOx5Fq5bF1ZzIruz76+E=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  pythonImportsCheck = [ "serverfiles" ];
+
+  meta = {
+    description = "An utility that accesses files on a HTTP server and stores them locally for reuse";
+    homepage = "https://github.com/biolab/serverfiles";
+    license = [ lib.licenses.gpl3 ];
```

```suggestion
    license = lib.licenses.gpl3Plus;
```
https://github.com/biolab/serverfiles/blob/00f60a9f4a841b5aae54f678d8bebfb0161dbb7b/LICENSE.txt#L9C40-L9C40